kube_scheduler/datadog_checks/kube_scheduler/data/auto_conf.yaml

```diff
@@ -1,16 +1,36 @@
+## @param ad_identifiers - list of strings - required
+## A list of container identifiers that are used by Autodiscovery to identify
+## which container the check should be run against. For more information, see:
+## https://docs.datadoghq.com/agent/guide/ad_identifiers/
+#
 ad_identifiers:
   - kube-scheduler
 
+## All options defined here are available to all instances.
+#
 init_config:
 
+## Every instance is scheduled independent of the others.
+#
 instances:
+
     ## @param prometheus_url - string - required
     ## The URL where your application metrics are exposed by Prometheus.
     #
-  - prometheus_url: "http://%%host%%:10251/metrics"
+  - prometheus_url: http://%%host%%:10251/metrics
 
-    ## @param bearer_token_auth - string - optional
+    ## @param bearer_token_auth - boolean - optional - default: true
     ## Used if you are using RBACs and need the Agent to authenticate
     ## against the APIServer to retrieve metrics. Default to true.
     #
-    bearer_token_auth: true
+    # bearer_token_auth: true
+
+    ## @param bearer_token_path - string - optional - default: /var/run/secrets/kubernetes.io/serviceaccount/token
```

Comment:
Why are we adding those few options here but not others? Should we leave it as it is right now as the auto_conf file is supposed to work out of the box or don't work at all ?

Reply:
Primarily to be in sync with `kube_apiserver` (https://github.com/DataDog/integrations-core/pull/7614), though there may be good reasons the options weren't included in this config.